### PR TITLE
Add experiment tracking to features section

### LINKED ISSUE #65: Add Experiment Tracking as a feature on the website
## Description

We don't mention Experiment Tracking at all on kedro.org. This would be the design the experiment tracking feature module on kedro.org.

### PR DESCRIPTION
**Pull request by tynandebold**
Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Resolves #65.

## Development notes

I took a screenshot of our demo page and added experiment tracking as the last item in the features section. 

## QA notes

This is what it looks like:

image

Take a look at the deploy preview URL for a closer look.

@stichbury would love your guidance for copy and @comym yours for design.

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Added tests to cover my changes, where applicable

### CODE REVIEW

**Comment by comym** [comment]
I'm speaking with @Mackay031 and we'll try to prepare an image for that.
Great to see it's all in place.

Will come back to you on that.

thanks

File: components/features/features-content.tsx [hunk 1]
```diff
@@ -136,4 +137,15 @@ export const hiddenContent: FeatureProps[] = [
...
   },
+  {
+    altText: 'Experiment Tracking example',
+    assetPosition: 'right',
+    buttonLink:
+      'https://kedro.readthedocs.io/en/latest/logging/experiment_tracking.html',
+    buttonText: 'Enable experiment tracking',
+    imageSrc: expTracking,
+    subtitle:
+      'Experiment tracking is a way to record all information that you would need to recreate and analyse a data science experiment.',
```
Comment (stichbury) [inline components/features/features-content.tsx:148]:
```suggestion
      'Experiment tracking records all the information that you need to recreate and analyse a data science experiment.',
```

**Review by stichbury — APPROVED** [review]
LGTM -- I've added a suggested revision. 

At some point, we should review all the copy for keyword SEO but for now, let's just get this nice change in.

**Comment by tynandebold** [comment]
@comym I've added your image. Please take a look: https://deploy-preview-69--kedro-website.netlify.app/

**Review by yetudada — APPROVED** [review]
This looks fantastic. I really, really love the simplified image @comym! Well done team!

**Review by comym — APPROVED** [review]

**Pull request merged**